docs/install/containers/recipes/index.md

```diff
@@ -149,6 +152,13 @@ In RelStorage mode, pass the connection DSN.
 docker run -e RELSTORAGE_DSN="dbname='plone' user='plone' host='db' password='password' port='5432'" pack
 ```
 
+In Docker Swarm, if the database is only available on an internal network, it's required to specify the network.
+For this situation, it might be easier to use `docker exec` to run the pack command in an existing `plone-backend` container.
+
+```shell
+docker exec RELSTORAGE_DSN="dbname='plone' user='plone' host='db' password='password' port='5432' network='internal" pack
```

Comment:
```suggestion
docker exec RELSTORAGE_DSN="dbname='plone' user='plone' host='db' password='password' port='5432' network='internal" backend pack
```

The above command assumes that the service that runs the Plone instance is named `backend`.
Otherwise replace `backend` with your container's name.

Reply:
I missed the container name in the command, so we need to add it.

We need to signal